src/index.ts

```diff
@@ -283,9 +328,17 @@ const translate = async (
               chalk`├── {red.bold ${file.name} is no longer used and will be deleted.}`,
             );
 
-            fs.unlinkSync(path.resolve(evaluateFilePath(workingDir, dirStructure, language), file.name));
+            fs.unlinkSync(
+              path.resolve(
+                evaluateFilePath(workingDir, dirStructure, language),
+                file.name,
+              ),
+            );
 
-            const cacheFile = path.resolve(evaluateFilePath(workingDir, dirStructure, language), file.name);
+            const cacheFile = path.resolve(
+              evaluateFilePath(workingDir, dirStructure, language),
+              file.name,
+            );
```

Comment:
It seems like you're using different formatting settings, or maybe Prettier was misconfigured on my end. If possible, it would be nice if the PR only touched lines that are actually changed. That way it's easier to inspect what's changed :)